pom.xml

```diff
@@ -4,13 +4,18 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>net.sf</groupId>
+    <groupId>livongo</groupId>
     <artifactId>log4jdbc4</artifactId>
-    <version>1.3</version>
+    <version>1.3.1</version>
     <packaging>jar</packaging>
     <name>log4jbdc4</name>
     <url>https://github.com/arthurblake/log4jdbc</url>
 
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
This should be referring JDK 11.

Reply:
Taking into account how old the code is, it does not matter because there are no language features requiring newer versions of Java. This change was made to make the code compile with JDK 11 I have. According to [`javac` doc](https://docs.oracle.com/en/java/javase/11/tools/javac.html#GUID-AEEC9F07-CB49-4E96-8BC7-BCC2C7F725C9), starting from JDK 9 `source` can only be 6 and higher:
> Beginning with JDK 9, javac no longer supports -source release settings less than or equal to 5. If settings less than or equal to 5 are used, then the javac command behaves as if -source 6 were specified.